common/djangoapps/util/course.py

```diff
@@ -5,24 +5,30 @@
 from django.conf import settings
 
 from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 log = logging.getLogger(__name__)
 
 
 def get_lms_link_for_about_page(course_key):
     """
-    Returns the url to the course about page.
+    Arguments:
+        course_key: A CourseKey object identifying the course.
+
+    Returns the course sharing url, this can be one of course's social sharing url, marketing url, or
+    lms course about url.
     """
     assert isinstance(course_key, CourseKey)
 
-    if settings.FEATURES.get('ENABLE_MKTG_SITE'):
-        # Root will be "https://www.edx.org". The complete URL will still not be exactly correct,
-        # but redirects exist from www.edx.org to get to the Drupal course about page URL.
-        about_base = settings.MKTG_URLS['ROOT']
+    course_overview = CourseOverview.objects.get(id=course_key)
```

Comment:
@nasthagiri , should we also respect feature flags like `ENABLE_MKTG_SITE `, `CUSTOM_COURSE_URLS `?